app/main.py

```diff
@@ -195,10 +213,11 @@ def change_layer_style(n_clicks=None):
 
 @app.callback(
     Output('store_live_alerts_data', 'data'),
-    [Input('update_live_alerts_data_workflow', 'children'),
-     Input('update_live_alerts_data_erase_buttons', 'children')]
+    [Input('update_live_alerts_data_workflow', 'data'),
+     Input('update_live_alerts_data_erase_buttons', 'data')],
+    State('update_live_alerts_data_workflow', 'data')
```

Comment:
There must be smth but I don't see the need of having `update_live_alerts_data_workflow` both as Input and State here ?

Reply:
You're right, I don't know why it was there in the first place!